README.md

```diff
@@ -827,6 +875,19 @@ colord("#cd853f").mix("#eee8aa", 0.6).toHex(); // "#e3c07e"
 colord("#008080").mix("#808000", 0.35).toHex(); // "#50805d"
 ```
 
+Also, plugin provides special mixtures such as [tints, shades, and tones](https://en.wikipedia.org/wiki/Tints_and_shades):
+
+<div align="center">
+<image src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Tint-tone-shade.svg/320px-Tint-tone-shade.svg.png" alt="tints, shades, and tones mixtures">
+</div>
+
+```js
+colord("#ff0000").tints(10).map(color => color.toHex()) // ["#ff0000", "#ff3f20", "#ff5c39", "#ff7551", "#ff8b68", "#ff9f80", "#ffb399", "#ffc6b2", "#ffd9cb", "#ffece5", "#ffffff"];
+colord("#ff0000").shades(10).map(color => color.toHex()) // ["#ff0000", "#e31004", "#c81707", "#ad1b09", "#931c0b", "#7a1b0b", "#621a0b", "#4a1709", "#341306", "#200d03", "#000000"];
+colord("#ff0000").tones(10).map(color => color.toHex()) // ["#ff0000", "#f52d14", "#ea4023", "#df4e30", "#d4583b", "#c86147", "#bc6952", "#af705e", "#a17669", "#917b75", "#808080"];
```

Comment:
I guess we can generate 4-5 colors here to keep the docs more readable. What do you think?

Reply:
Sure, no need to have here so many colors :)